package.json

```diff
@@ -102,22 +102,21 @@
     "@types/xml2js": "0.4.9",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
-    "dd-trace": "0.36.6",
+    "dd-trace": "3.5.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-typescript": "2.7.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-no-null": "^1.0.2",
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-prettier": "4.0.0",
-    "jest": "27.0.5",
-    "jest-environment-node": "27.0.5",
+    "jest": "28.1.3",
```

Comment:
I couldn't bump to the latest major (29) because it requires a bump to `ts-jest`, which requires a bump to `typescript` which would break our linter right now